TestsFoundation/TestsFoundation.xcodeproj/project.pbxproj

```diff
@@ -1136,9 +1136,11 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.instructure.TestsFoundation;
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = NO;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_TREAT_WARNINGS_AS_ERRORS = NO;
 				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
I just removed these, are they necessary?

Reply:
No idea, I just let xcode call the shots.